Source/Core/Core/PowerPC/PowerPC.cpp

```diff
@@ -34,7 +34,7 @@ PowerPCState ppcState;
 
 static CPUCoreBase* s_cpu_core_base = nullptr;
 static bool s_cpu_core_base_is_injected = false;
-Interpreter* const s_interpreter = Interpreter::getInstance();
+Interpreter interpreter;
```

Comment:
Dolphin prefixes static / global variables with `s_`, and that should be kept here. (The change of type from a pointer to a value might be confusing, but I think this is important enough to keep consistent.)